cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,21 @@
 /// <reference types='cypress' />
+import { LaptopsTab } from '../support/pages/customerUserWay';
 
-describe('', () => {
-  before(() => {
-
+const Laptops = new LaptopsTab();
+describe('Userflow using POM', () => {
+  before('open webpage', () => {
+    Laptops.navigateToHome();
   });
 
-  it('', () => {
-
+  it('full userway', () => {
```

Comment:
test name should start with word "should"
Describe name (Feature) + should .......